packages/snowpack/src/commands/build.ts

```diff
@@ -268,14 +282,21 @@ export async function command(commandOptions: CommandOptions) {
   });
 
   if (config.buildOptions.minify) {
-    console.log(colors.yellow('! minifying javascript...'));
+    const minifierStart = performance.now();
+    console.log(colors.yellow('! optimizing for production...'));
```

Comment:
I don't like the idea of calling this `minify` in the docs, but then logging it as a more general `optimize`. Happy to solve this by changing either the output or the config name.